examples/models/latex_labels.py

```diff
@@ -10,7 +10,7 @@
 from bokeh.models import Label
 from bokeh.plotting import figure, show
 
-p = figure(width=600, height=600, x_range=(0, 10), y_range=(0, 10))
+p = figure(width=600, height=600, x_range=(0, 20), y_range=(0, 20))
```

Comment:
Here is the plot how it looked before and after your changes.

|`figure(width=600, height=600, x_range=(0, 10), y_range=(0, 10))`|`figure(width=600, height=600, x_range=(0, 20), y_range=(0, 20))`|
|--|--|
|![grafik](https://github.com/bokeh/bokeh/assets/68053396/2c7d6be0-f569-47f1-9b57-e99436bb3364)|![grafik](https://github.com/bokeh/bokeh/assets/68053396/38fe867e-a371-4019-85ef-60bc90a2f221)|

I think this is a mistake. Please revert this change.

Reply:
Yeah! I guess I overlooked that